### DISCUSSION TITLE
Getting mouse position in window not working with the old fill-window example

### DISCUSSION THREAD

**Discussion by SwagDeNoli** [parasyte/pixels#364] [Q&A] [answered]
Noticed while using the fill-window renderer mouse position is incorrect 
Modified example code: https://gist.github.com/SwagDeNoli/8724152fa782f11d8907b96d511816a8
The pixels/winit mouse position code is off (possibly due to the custom renderer?)

**Comment by parasyte** [comment, marked as answer]
Possibly caused by the custom renderer. I'd have to revisit that PR to be sure, but thank you for reporting the issue!

**Comment by SwagDeNoli** [comment]
I'm looking forward to that. For the time being I'll start using the default renderer + update to the more recent version, as I can workaround not using it for the moment.
